fastlane/amplify_pods.rb

```diff
@@ -0,0 +1,63 @@
+module AmplifyPods
```

Comment:
How are we keeping these in sync with newly-added targets?

Reply:
At the moment, this list would need to get updated any time a target is added